### PR TITLE
update controller deploy ref

### PR DESCRIPTION
**Pull request by qrkourier**

### CODE REVIEW

**Comment by qrkourier** [comment]
@dovholuknf Feedback from @emoscardini was move the "Bring Your Own State" section to a separate page instead of cramming everything into one page.

File: docusaurus/docs/downloads.mdx [hunk 1]
```diff
@@ -99,6 +99,19 @@ curl -sS https://get.openziti.io/install.bash \
 
 <br/>
 
+### Install the `ziti-controller.service`
+
```
Comment (dovholuknf) [inline docusaurus/docs/downloads.mdx:103]: let's move this info to the "deployment reference" and leave a link at the bottom of this page stating: "If you're trying to deploy an OpenZiti overlay network, follow the deployment reference guide here"

File: docusaurus/docs/reference/deployments/01-controller.md [hunk 2]
```diff
@@ -6,24 +6,126 @@ sidebar_label: Controller
 
 import CliLogFormats from '../../_cli-log-levels-and-formats.md'
 
-This article provides some guidance for deploying a Ziti Controller. You can go back and read [the controller component introduction](/learn/introduction/03-components.md#openziti-controller) if needed.
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide. [Kubernetes is covered separately](/guides/kubernetes/hosting/kubernetes-controller.mdx).
```
Comment (dovholuknf) [inline docusaurus/docs/reference/deployments/01-controller.md:9]: There's no need for the kubernetes covered separately comment. Let them discover via the sidebar

File: docusaurus/docs/reference/deployments/01-controller.md [hunk 3]
```diff
@@ -6,24 +6,126 @@ sidebar_label: Controller
...
 import CliLogFormats from '../../_cli-log-levels-and-formats.md'
 
-This article provides some guidance for deploying a Ziti Controller. You can go back and read [the controller component introduction](/learn/introduction/03-components.md#openziti-controller) if needed.
+This article is about deploying a Ziti Controller as a Linux system service. [The controller introduction](/learn/introduction/03-components.md#openziti-controller) introduces the architectural concepts used in this guide. [Kubernetes is covered separately](/guides/kubernetes/hosting/kubernetes-controller.mdx).
 
-## Public Key Infrastructure
+## Installation
 
-You will need a PKI setup for Ziti. If you follow one of the [quickstart guides](/learn/quickstarts/network/hosted.md) then a PKI will be generated for you. This is probably the best way to start out. Pay particular attention to the PKI-related fields in the configuration file, and check out [the pki page](/learn/core-concepts/pki.md).
+The controller package is installed the same way as the `ziti` CLI and depends on the `openziti` package the provides the CLI.
```
Comment (dovholuknf) [inline docusaurus/docs/reference/deployments/01-controller.md:13]: This is the right place to put that installaction stuff from above. I'd remove most of the words here and just show them how to install it

File: docusaurus/docs/reference/deployments/01-controller.md [hunk 4]
```diff
@@ -6,24 +6,126 @@ sidebar_label: Controller
...
 
-## Configuration
+The signed package sources are found [in the Linux downloads tab](/downloads.mdx?os=Linux).
```
Comment (dovholuknf) [inline docusaurus/docs/reference/deployments/01-controller.md:15]: no longer relevant

**Comment by dovholuknf** [comment]
On the whole, there feels like a fair bit of reference-type-words mixed throughout that probably don't belong in a guide-type document. Would you go through and find anything that feels reference-ish and unecessary and remove? IPC Agent is not necessary to understand, SNI and ALPN seem to be reference-related not guide-related to me. There might be other stuff that could be trimmed out as well but as I read through those sorts of things are striking me as not really being inline with what a 'guide' is. some bit of it, is necessary but this should focus more bringing it up and how, not the details under it. it does make me feel like there's some kind of related reference material here that could be exctracted and moved to reference

File: docusaurus/docs/reference/deployments/01-controller.md [hunk 5]
```diff
@@ -6,24 +6,126 @@ sidebar_label: Controller
...
+
+### Bootstrapping the Database
+
+The controller requires a BoltDB database to store its state. The Linux system service will initialize a database with a default admin password during the first startup, unless one already exists.
+
+You must specify the management password for the default admin user before starting the service. This is done by setting `ZITI_PWD` in `/opt/openziti/etc/controller/env` or one of `LoadCredential` or `SetCredential` in `/lib/systemd/system/ziti-controller.service`. You may delete the password after bootstrapping for security.
+
+Disable bootstrapping the database by setting `ZITI_BOOTSTRAP_DATABASE=false` in `/lib/systemd/system/ziti-controller.service`.
+
+## Bring Your Own State
```
Comment (dovholuknf) [inline docusaurus/docs/reference/deployments/01-controller.md:57]: let's extract this to it's own page and lets focus on not making this a generic example, but focus specifically on "migrating a quickstart"

**Comment by qrkourier** [comment]
aggregating changes in #820

**Pull request closed without merging**